### PR TITLE
Recommend Blazor for UI in .NET 9

### PR DESCRIPTION
**Pull request by Rick-Anderson**
---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/tutorials/choose-web-ui.md](https://github.com/dotnet/AspNetCore.Docs/blob/270cac4eb60f3464befeae803a1cb31b74a872c2/aspnetcore/tutorials/choose-web-ui.md) | [Choose an ASP.NET Core web UI](https://review.learn.microsoft.com/en-us/aspnet/core/tutorials/choose-web-ui?branch=pr-en-us-33372) |

### CODE REVIEW

**Comment by guardrex** [comment]
I'm a bit confused because the Blazor section already states it for >=8.0.

**Comment by Rick-Anderson** [comment]
> I'm a bit confused because the Blazor section already states it for >=8.0.

I'd like to say it right away. It says that in the Blazor section and some folks will skip that section because they haven't heard that much about Blazor. Is it too much to say it twice? Would it be better to remove the 2nd recommendation?

 Blazor is a full-stack web UI framework ~and is recommended for most web UI scenarios.~

Recommended for what? I think saying it at the top makes it more clear.

**Comment by guardrex** [comment]
Yes, I recommend removing the second mention.

Do you want to version this? It kind'a breaks with the Version-by-INCLUDE approach, and I don't think it matters that much if it says for 8.0 that Blazor is favored. I think that's what DR told me ... that he was pushing it as the primary app framework for 8.0.

**Comment by Rick-Anderson** [comment]
> Do you want to version this? It kind'a breaks with the Version-by-INCLUDE approach, and I don't think it matters that much if it says for 8.0 that Blazor is favored. I think that's what DR told me ... that he was pushing it as the primary app framework for 8.0.

File: aspnetcore/tutorials/choose-web-ui.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ uid: tutorials/choose-web-ui
 :::moniker range=">= aspnetcore-8.0"
 
 ASP.NET Core is a complete UI framework. Choose which functionalities to combine that fit the app's web UI needs.
+For new project development, we recommend ASP.NET Core Blazor.
```
Comment (Rick-Anderson) [inline aspnetcore/tutorials/choose-web-ui.md:15]:
```suggestion

For new project development, we recommend ASP.NET Core Blazor.
```

**Review by guardrex — APPROVED** [review]

**Pull request merged**